docs/test_lab.md

```diff
@@ -99,9 +99,9 @@ Done with physical switch configuration.
 
 ## Common Errors
 
-The following message in `inst/faucet.log` indicates that a switch is trying
+The message below, in `inst/faucet.log`, indicates that a switch is trying
 to connect to faucet, but `ext_dpid` is configured wrong: simply copy/paste
-the hex dipd (e.g. _0x1aeb960541_) into the config file.
+the hex dipd (e.g. _0x1aeb960541_) from the `inst/faucet.log` into the `local/system.conf` file.
```

Comment:
==> from `inst/faucet.log` into `local/system.conf`.

Reply:
You missed the first "the"

Reply:
Thanks. A bit difficult on the bus, this thing is bouncing all around, can hardly see the screen.